src/lib/evil/evil_dlfcn.h

```diff
@@ -143,9 +143,11 @@ struct Dl_info
  * @ingroup Evil_Dlfcn
  */
 EAPI void *dlopen(const char* path, int mode);
+
```

Comment:
why?

Reply:
I don't like two different things (a declaration and a preprocessor definition) to be so close.